srcpkgs/codelite/template

```diff
@@ -33,9 +30,3 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 CXXFLAGS="-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
-
-post_extract() {
-	mv codelite-${version}/* .
-	rm -rf universal-ctags
-	mv ctags-52c724d1132d78ea44894bfe2eaca44f38a9bd85 universal-ctags
-}
```

Comment:
Since the release tarball includes the ctags submodule, we no longer need to do this